reconciletags/examples/README.md

```diff
@@ -0,0 +1,116 @@
+Running Retagger with Kubernetes
+================================
+The retagger is a tool used to tag images. You will need a git repository which includes a json file in [this](https://github.com/GoogleCloudPlatform/runtimes-common/blob/master/reconciletags/sample.json) format, which contains all the images and associated tags that you want to retag. Then, with Kubernetes, you can create a Pod to retag those images once, or a CronJob which will run the retagger at scheduled intervals. The benefit of this is that if a change is made to the json file in your git repository, the CronJob will automatically retag your images at the next scheduled time.
+
+Getting Started
+---------------
+
+To run the retagger through Kubernetes as either a Pod or a CronJob on a Google Container Engine cluster, please follow these steps:
+
+__Creating a Service Account__
+
+First, create a service account, which will later be used for authentication purposes.
+
+![alt text](images/image1.png)
+
+Name the account, and select `Storage > Storage Admin` as the role. Download as a JSON file, and rename this file to **retagger_secret.json**
+
+__Create a Secret__
+
+A Kubernetes secret will be used to authenticate to GCR, allowing the retagger to pull and push images. To create a secret named retagger-secret, run the following kubectl command:
+
+`kubectl create secret generic retagger-secret  --from-file=[PATH TO retagger_secret.json]`
+
+To ensure the secret was created, run
+
+`kubectl get secrets`
+
+retagger-secret should be in the output list of secrets.
+
+__Activate Service Account in gCloud__
```

Comment:
I think they prefer to refer to it as "the Cloud SDK"

Reply:
Actually though, I don't think you need this step. It was just for permissions and debugging.